data_model.mdd

```diff
@@ -370,8 +372,13 @@
     SupportRecordLog {
         int id PK
         int support_record_id FK "Support Record ID"
-        entry_date date "Support Record Log Entry Date"
-        comments string(MAX) "Support Record Log Entry details"
+        date entry_date "Support Record Log Entry Date"
+        string(MAX) comments "Support Record Log Entry details"
+    }
+    SupportRecordIssues {
```

Comment:
Hey @aunroel, I'm not sure this is needed anymore from our conversation day?

Reply:
this is still needed

we were talking about Training Support status


this one is about SupportCase issue type, which can be as many as you want 😅